### ISSUE TITLE
Fix wrong question when track media is not available

### ISSUE THREAD

**Issue by 4www** [internet4000/trivia-radio4000#11]
Sometimes the quiz is on a track which media cannot be played by the radio4000-player.

We should listen on the `mediaNotAvailable` event sent by the player, and refresh the question.

**Comment by oskarrough** [comment]
Yes. And initially filter out unavailable tracks in https://github.com/internet4000/trivia-radio4000/blob/master/app/routes/quiz/questions.js#L7
